admin/services/data-access/check-window.data.service.js

```diff
@@ -2,31 +2,76 @@
 
 const moment = require('moment')
 const CheckWindow = require('../../models/check-window')
+const winston = require('winston')
+const sqlService = require('./sql.service')
+const TYPES = require('tedious').TYPES
 
 const checkWindowDataService = {
   /**
    * Fetch check window document by id.
    * @param id
+   * @deprecated use sqlFetchCheckWindow
    * @returns {Promise.<*>}
    */
   fetchCheckWindow: async (id) => {
+    winston.warn('check-window.data.service.fetchCheckWindow is deprecated')
     return CheckWindow.findOne({'_id': id, 'isDeleted': false}).exec()
+  },
+    /**
+   * Fetch check window document by id.
+   * @param id
+   * @returns {Promise.<*>}
+   */
+  sqlFetchCheckWindow: async (id) => {
```

Comment:
it might be better to be called `sqlFindCheckWindow` for consistency with other data access services.